applications/ShapeOptimizationApplication/test_examples/05_Shell_Opt_Using_Trust_Region_Approach/run_optimization.py

```diff
@@ -0,0 +1,57 @@
+# Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
about the naming of this folder:
`Using` and `Approach` do not really add more information, they just make the path longer =)